### PR TITLE
DOC: added reference and example in jacobi docstring

### PR DESCRIPTION
**Pull request by lorenzophys**
#### Reference issue

https://github.com/scipy/scipy/issues/7168

#### What does this implement/fix?

The jacobi polynomials docstring in scipy.special didn't have any example. I added a couple of them with a reference

### CODE REVIEW

**Comment by person142** [comment]
Nice additions, thanks @lorenzophys!

File: scipy/special/orthogonal.py [hunk 1]
```diff
@@ -317,6 +317,44 @@ def jacobi(n, alpha, beta, monic=False):
...
+    >>> from scipy.special import jacobi
+    >>> x = np.arange(-1.0, 1.0, 0.01)
+    >>> np.allclose(jacobi(0, 2, 2)(x), jacobi(1, 2, 1)(x) - jacobi(1, 1, 2)(x))
+    True
+
+    Plot of the Jacobi polynomaial :math:`P_5^{(\alpha, -0.5)}` for
+    different values of :math:`\alpha`:
+
+    >>> import matplotlib.pyplot as plt
+    >>> import numpy as np
```
Comment (tylerjereddy) [inline scipy/special/orthogonal.py:347]: minor: I think this import is implicit, unless we recently changed our policy on that

Comment (lorenzophys) [reply scipy/special/orthogonal.py:347]: Also the hermite docstring imports numpy this way, and I took that one as an example. I just checked and the docs build normally without the import. Should I open a separate PR or can I include this fix in my next one?

Comment (tylerjereddy) [reply scipy/special/orthogonal.py:347]: Probably not worth its own PR for now, just something to keep in mind perhaps

**Pull request merged**